.gitignore

```diff
@@ -2,4 +2,13 @@
 /target
 /test
 /out
-dependency-reduced-pom.xml
\ No newline at end of file
+dependency-reduced-pom.xml
+plugins/PArekPlugin.jar
+lib/verifier.jar
+lib/DarkFlash.dll
+lib/DarkBot.dll
+lib/DarkBoatAPI.dll
+DarkBOT.iml
+credentials.json
+config_old.json
+config.json
```

Comment:
Keep this file as-is.
Set your working directory in your IDE run configurations to point to an outside folder with all the release stuff